### ISSUE TITLE
Hive feedback from todays attempt

### ISSUE THREAD

**Issue by BobdaBiscuit** [d3athrow/vgstation13#14560]
Just like to preface this with I very much enjoyed what I got to see of the Hive and I like it's challenge, but it seems like most crews won't be able to make it through without something gamebreaking, the rest of this is just some opinions and feedback.

>DEAD: Pocket Buddy cries, "Hard is onething but this is a little bit too far towards the old untested SNES knockoff level"

>Dead Crewmen: 65 (-16250 Points)

So I solo'd about half of the way in (before any of the objectives) as an angry diona with a claymore/guns and watched the rest of the crew attempt it after my death, it says enough about the away mission that I alone made it the farthest.

An attack party of around 4-7 people went in armed to the teeth with gear, got about half of the way I did and had to retreat to the medical bay they constructed at the start of the hive. By the time they got back everything respawned and history repeated itself, only with corpses starting to become unrecoverable.

The whole away mission is one huge endurance run of pure death, with no apparent breaks or havens, your progress also resets every couple of minutes so you can't get back safely in most cases. Parties can be doing well then walk into a room of 3 defenders, then get utterly decimated as all the hallways are 1 tile big and everyone gets trampled.

What ended up happening was the station started to colonise the hive, starting from the gateway, atmos systems and a medbay built since there was no hope of progressing before promptly giving up and continuing the round as normal now lacking half of the crew.

So let's look at the issues from what I can see:

* Endless hordes of enemies with respawns
* Endless hordes of defenders beating your shit in
* Endless hordes of defenders spawning as soon as you clear the defenders from the previous point out
* Lack of a safe midway refuge/breaks/resupply areas
* Lack of built in methods of healing
* Tiny cramped hallways which impede group progress

Now for some solutions that were suggested, these are just rough ideas feel free to shit on them.

* "Safe rooms" ala left4dead. Have a couple of them about as kinda check points, if you want it real ez have them linked to the gateway so you can get to and from them fast when you find new ones. These rooms should be free of monsters and have enough facilities and resources to restock attack parties that made it there, rechargers/meds etc.

* The asteroid method, have some meme like hivelord cores for the hive. Some item that magically heals damage you've taken that can be harvested from something in the hive, could be useful for when literally all your bones get broken.

* Open the place up a bit, make sure corridors are atleast 2 tiles wide so multiple people can attack oncoming enemies.

* Increase the mob respawn time. Having defenders and other mobs spawn on your ass endlessly gets real old, especially as the attack party gets more and more wounded.

tl;dr
2hard pls nerf k thx

**Comment by coldcola** [comment]
yes please make it easier to get the game-breaking loot that is in this away mission thxbai

**Comment by ReindH** [comment]
Maybe make the gateway area larger and easily defendable so med personnel and engineers get their fix.
Less spawns but the same amount of hostiles so there is an actual feeling of progress after you first cleared an area. That's all I can think of.

**Comment by BobdaBiscuit** [comment]
Yeah the initial amounts of baddies seems fine/challenging.

**Comment by Kurfursten** [comment]
It's not so much that it was too challenging as that it just wasn't fun. This mission felt like a slog to me. Rafid gets me excited and I'm always up for exploring it, but this was my first Hive mission and I left all my gear behind for other people and logged out. Constant 1-wide mazes with slowfloors and enemies that are hard to see because of their colors on the floor just make it all a rather unpleasant mission.

It's one thing to have environmental hazards in set locations (like Rafid's fire rooms) but this thing is just one big marsh of unpleasant, and combined with the total lack of checkpoints or any way to make meaningful progress that won't reset, it feels very pointless unless you somehow have gear to breeze through it all in one go.

**Comment by Probe1** [comment]
@unid15 

also it was supposed to be too hard.  you guys forgot what away missions were like before you all learned how to break them down to the last and get through.  

remember emitter hell before you realized you could just unwrench a girder and cheese it?

**Comment by BobdaBiscuit** [comment]
Couple other ideas for potential nerfs for giggles.

* Respawned mobs have less heath/damage
* IC warning before a new wave of mobs spawn
